pysiaf/tests/test_match_v2v3.py

```diff
@@ -0,0 +1,29 @@
+from pysiaf import iando
+from pysiaf.utils import tools
+
+
```

Comment:
Good point, that I had missed. Please define this as a function, e.g. 
`def test_match_v2v3_nircam():`